defaults.go

```diff
@@ -154,8 +155,8 @@ func mustToRawJson(v interface{}) (string, error) {
 }
 
 // ternary returns the first value if the last value is true, otherwise returns the second value.
-func ternary(vt interface{}, vf interface{}, v bool) interface{} {
-	if v {
+func ternary(vt interface{}, vf interface{}, v interface{}) interface{} {
+	if truth, ok := template.IsTrue(v); ok && truth {
```

Comment:
In my own library, I'd implemented this with a panic for `!ok` (which I'm happy to adjust this to), but I wasn't sure if `panic` was appropriate here. :sweat_smile: